plugins/module_utils/foreman_helper.py

```diff
@@ -1011,6 +1011,9 @@ def _entity_spec_helper(spec):
     """
     entity_spec = {'id': {}}
     argument_spec = {}
+
+    _FILTER_SPEC_KEYS = ['thin', 'scope', 'resource_type', 'search_by', 'search_operator', 'resolve', 'ensure']
```

Comment:
@bagasse @mdellweg is this list complete? :)

Reply:
Do we need `failsafe`?
I fear, i have lost track about the evolution of entity_spec.

Reply:
@mdellweg it works without 🤷